Project_FreeRTOS/main/task_khoi.h

```diff
@@ -9,11 +9,11 @@ void task_KHOI(void *pvParameters)
   for(;;)
   {
     Message mq_send = {0};
-      mq_send.id_Tx = OUT_FAN;
-      mq_send.id_Rx = IN_MQ_135;
+      mq_send.id_Tx = IN_MQ_135;
+      mq_send.id_Rx = OUT_FAN;
       mq_send.payload = FAN_MQ_DISABLE;
-      xQueueSend(Platform_Queue, &mq_send, portMAX_DELAY);
-      if(xQueueSend(Platform_Queue, &mq_send, portMAX_DELAY) == pdPASS)
+      xQueueSend(Khoi_Queue, &mq_send, portMAX_DELAY);
```

Comment:
This line of is redundant, please remove it

Reply:
completed